App/src/main/java/cc/softwarefactory/lokki/android/activities/MainActivity.java

```diff
@@ -201,13 +202,16 @@ private void checkIfUserIsLoggedIn() {
 
     @Override
     public void onNavigationDrawerItemSelected(int position) {
+        // Position of the logout button
+        final int logoutPos = 5;
```

Comment:
I am not sure placing the logout button as an item in the navigation drawer is the right place for it as it is an action rather than a "section"...